### PR TITLE
Edge to edge HLS player

### PR DESCRIPTION
**Pull request by ygit**
# Description

- Made the HLS player edge to edge 

### Pre-launch Checklist

- [x] The [Documentation] is updated accordingly, or this PR doesn't require it.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making, or this PR is test-exempt.
- [x] All existing and new tests are passing.

[Documentation]: https://www.100ms.live/docs

### CODE REVIEW

File: packages/react-native-room-kit/example/ios/RNExample/Info.plist [hunk 1]
```diff
@@ -89,10 +89,11 @@
 	<key>UIStatusBarHidden</key>
 	<true/>
 	<key>UIStatusBarStyle</key>
-	<string></string>
+	<string>UIStatusBarStyleDarkContent</string>
```
Comment (stanwolverine) [inline packages/react-native-room-kit/example/ios/RNExample/Info.plist:92]: will this give us dark content on status bar? @ygit

File: packages/react-native-hms/ios/HMSHLSPlayerManager.swift [hunk 2]
```diff
@@ -200,6 +200,13 @@ class HMSHLSPlayer: UIView {
         self.addSubview(playerViewController.view)
 
         attachPlayerPlaybackListeners()
+        
+        hmsHLSPlayerViewController?.showsPlaybackControls = false
```
Comment (stanwolverine) [inline packages/react-native-hms/ios/HMSHLSPlayerManager.swift:204]: is this still configurable from JS side? @ygit

Comment (ygit) [reply packages/react-native-hms/ios/HMSHLSPlayerManager.swift:204]: No

File: packages/react-native-room-kit/example/ios/RNExample/Info.plist (hunk 1 above)
Comment (ygit) [reply packages/react-native-room-kit/example/ios/RNExample/Info.plist:92]: yes

**Review by stanwolverine — APPROVED** [review]

**Pull request merged**